styles/card.css

```diff
@@ -0,0 +1,118 @@
+.card {
+    border: 1px solid var(--theme900);
+    width: 300px;
+    padding: 1rem;
+    margin: 1rem;
+    line-height: 1.5;
+    text-align: center;
+    background-color: var(--theme50);
+    border-top-left-radius: 1rem;
+    border-bottom-right-radius: 1rem;
+    box-shadow: 2px 2px 10px var(--theme300);
+    position: relative;
+}
+
+.card-horizontal {
+    border: 1px solid var(--theme900);
+    display: grid;
+    grid-template-areas:
+        "pic header"
+        "pic desc"
+        "pic btn";
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-rows: repeat(3, 1fr);
```

Comment:
Here flex would be better it think

Reply:
Here, we would have a flex inside flex right?
One flex to keep the image and text container next to each other.
Second flex to align the contents within the text-container. To avoid this, I used a single grid.